src/main/java/CliUI.java

```diff
@@ -110,4 +179,20 @@ void runCommandsOnArgs(String[] args) {
                 break;
         }
     }
+
+    int getIntInput() {
+        int input = 0;
+
+        try {
+            input = keyboard.nextInt();
+        } catch (InputMismatchException e) {
+            System.out.println("Please enter a number");
+        }
+        if (input < 1 || input > 7) {
```

Comment:
Maybe make these values a variable. When extending the code, for example adding or removing a function in the switch on line 85, it might not directly seem obvious that a line a hundred lines down in the code also needs to be updated.
Instead of this you could also add a comment at line 85 to at least notice the developer.

Reply:
Good observation, we should've used the Command pattern and use it's boundaries in this case.